requirements.txt

```diff
@@ -1,4 +1,3 @@
-django>=1.11,<2.2
```

Comment:
Wait, why did you remove django from requirements, it was fine before!
Since we discussed on gitter that `1.11` can be dropped. `django>=2.0,<2.3` would be fine!